extension/js/common/core/crypto/key.ts

```diff
@@ -50,16 +50,10 @@ export type PubkeyResult = { pubkey: Key, email: string, isMine: boolean };
 export type Contact = {
   email: string;
   name: string | null;
-  pubkey: Key | null;
+  pubkey: Key | undefined;
   has_pgp: 0 | 1;
```

Comment:
Since we are doing a migration anyway, could we rename these to be consistently camelCase? To use `hasPgp`, `lastUse` etc.

Reply:
This is a purely DTO class. We can of course refactor it in this PR, making it grow even bigger.